var/lib/delphix-platform/os-migration

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+MIGRATE_CONFIG_SCRIPT="/opt/delphix/migration/migrate_config.py"
```

Comment:
If this script is provided by the virtualization package, why wouldn't we have this service provided by the virtualization package too?

It seems a little strange that we'd have the script live in another repository than the service that'll execute that script, especially considering this service doesn't really do much beyond calling that script.

Reply:
Migration itself belongs to the platform / appliance rather than the virtualization component. This script and other migration scripts are placed into virtualization because some of its components (e.g. network migrator) are written in Java and have dependencies/have to be compiled with the app-stack code.

`/opt/delphix/migration/migrate_config.py` is to be called from various places (migration dx_ scripts, this service, manually for testing support bundles).

I can move this service to virtualization if you insist, but I personally don't think it would belong there more than it belongs in delphix-platform.

Reply:
The circular dependency is just a little awkward to me.. i.e. migration is provided by the platform package, but (implicitly) depends on the virtualization package.. and virtualization already depends on the platform package.

I understand the conceptual split, and the desire to have the concept of migration be a platform package concept, but practically the implementation makes me thing it's very much tied to the virtualization package.

Ultimately, I'm OK with how this is implemented (it'll work for customer systems either way), it just seems a little awkward and prone to break if the virtualization package isn't installed on the system being migrated.